lib/matplotlib/figure.py

```diff
@@ -750,19 +755,24 @@ def suptitle(self, t, *, x=.5, y=.98, **kwargs):
             sup.remove()
         else:
             self._suptitle = sup
-        if self._layoutbox is not None:
-            # assign a layout box to the suptitle...
-            figlb = self._layoutbox
-            self._suptitle._layoutbox = layoutbox.LayoutBox(
-                                            parent=figlb,
-                                            name=figlb.name+'.suptitle')
-            for child in figlb.children:
-                if not (child == self._suptitle._layoutbox):
-                    w_pad, h_pad, wspace, hspace =  \
-                            self.get_constrained_layout_pads(
-                                    relative=True)
-                    layoutbox.vstack([self._suptitle._layoutbox, child],
-                                     padding=h_pad*2., strength='required')
+            if self._layoutbox is not None:
+                if not manual_position:
+                    figlb = self._layoutbox
+                    self._suptitle._layoutbox = \
+                            layoutbox.LayoutBox(parent=figlb,
+                                                artist=self._suptitle,
+                                                name=figlb.name+'.suptitle')
+                    for child in figlb.children:
+                        if not (child == self._suptitle._layoutbox):
+                            w_pad, h_pad, wspace, hspace =  \
+                                    self.get_constrained_layout_pads(
+                                            relative=True)
+                            layoutbox.vstack([self._suptitle._layoutbox,
+                                                child],
+                                                padding=h_pad*2.,
+                                                strength='required')
+                else:
```

Comment:
Purely a matter of personal taste and not a blocker at all, but I would have rather written
```python
if manual_position:
    self._suptitle._layoutbox = None
else:
    figlb = ...
```
which I find a bit easier to read as in the other case, the long indented blocks may make it easy to misread the level indenting.

Reply:
Agreed that it seems more legible to move the manual_position case out first.